docs/python/testing.md

```diff
@@ -90,11 +90,13 @@ With this code, you can experience working with tests in VS Code as described in
 
 ## Enable a test framework
 
-Testing in Python is disabled by default. To enable testing, use the **Python: Configure Tests** command on the Command Palette. This command prompts you to select a test framework, the folder containing tests, and the pattern used to identify test files.
+Once you install the Python extension, a test beaker icon will be available on the VS Code activity bar -- that's the **Test Explorer**. When opening the test explorer, you will see a "Configure Tests" button if you don't have a test framework enabled. Once you click this button, you will be prompted you to select a test framework, the folder containing tests, and the pattern used to identify test files.
 
-You can also configure testing manually by setting *one and only one* of the following settings to true: `python.testing.unittestEnabled` or `python.testing.pytestEnabled`. Each framework also has specific configuration settings as described under [Test configuration settings](#test-configuration-settings) for their folders and patterns.
+[comment]: <TODO: add screenshot>
 
-It's important that you enable only a single test framework at a time. For this reason, when you enable one framework also be sure to disable the others. The **Python: Configure Tests** command does this automatically.
+You can configure your tests anytime by using  the **Python: Configure Tests** command on the Command Palette. You can also configure testing manually by setting either `python.testing.unittestEnabled` or `python.testing.pytestEnabled` to true. Each framework also has specific configuration settings as described under [Test configuration settings](#test-configuration-settings) for their folders and patterns.
+
+It's important that you enable only a single test framework at a time for a given workspace. For this reason, when you enable one framework also be sure to disable the others. The **Python: Configure Tests** command does this automatically.
```

Comment:
If you enable both `pytest` and `unittest`, we only run `pytest`. we should verify this once.